src/hints/vars.rs

```diff
@@ -86,4 +86,5 @@ pub mod ids {
 pub mod constants {
     pub const BLOCK_HASH_CONTRACT_ADDRESS: &str = "starkware.starknet.core.os.constants.BLOCK_HASH_CONTRACT_ADDRESS";
     pub const STORED_BLOCK_HASH_BUFFER: &str = "starkware.starknet.core.os.constants.STORED_BLOCK_HASH_BUFFER";
+    pub const VALIDATED: &str = "starkware.starknet.core.os.constants.VALIDATED";
```

Comment:
This var name is likely to conflict with a future string...

Reply:
:man_shrugging: